libr/debug/plugin.c

```diff
@@ -65,20 +80,22 @@ R_API bool r_debug_plugin_list(RDebug *dbg, int mode) {
 		}
 		pj_a (pj);
 	}
-	r_list_foreach (dbg->plugins, iter, h) {
-		int sp = 8 - strlen (h->meta.name);
+
+	RDebugPluginSession *ds;
+	R_VEC_FOREACH (dbg->plugins, ds) {
+		int sp = 8 - strlen (ds->plugin.meta.name);
 		spaces[sp] = 0;
 		if (mode == 'q') {
-			dbg->cb_printf ("%s\n", h->meta.name);
+			dbg->cb_printf ("%s\n", ds->plugin.meta.name);
 		} else if (mode == 'j') {
 			pj_o (pj);
-			pj_ks (pj, "name", h->meta.name);
-			pj_ks (pj, "license", h->meta.license);
+			pj_ks (pj, "name", ds->plugin.meta.name);
```

Comment:
at some point we can have a generic function that takes pj and PluginMeta and does this for all the plugin types

Reply:
That would be nice :)

Reply:
That would be nice :)